### ISSUE TITLE
Update package.json needed for working on Ubuntu 19 and Windows 10

### ISSUE THREAD

**Issue by tranKILL** [YogliB/svelte-fullcalendar#81]
Wasn't working on Ubuntu 19 and Windows 10 yesterday
With this package.json It works for me ("latest" of today for anything)

```
{
	"name": "svelte-fullcalendar-example",
	"devDependencies": {
		"node-sass": "4.13.0",
		"npm-run-all": "4.1.5",
		"rollup": "1.27.14",
		"rollup-plugin-commonjs": "10.1.0",
		"rollup-plugin-livereload": "1.0.4",
		"rollup-plugin-node-resolve": "5.2.0",
		"rollup-plugin-postcss": "2.0.3",
		"rollup-plugin-svelte": "5.1.1",
		"rollup-plugin-terser": "5.1.3",
		"svelte": "3.16.7",
		"svelte-preprocess": "3.3.0"
	},
	"dependencies": {
		"@fullcalendar/core": "4.3.1",
		"@fullcalendar/daygrid": "4.3.0",
		"@fullcalendar/interaction": "4.3.0",
		"@fullcalendar/resource-timeline": "4.3.0",
		"@fullcalendar/timegrid": "4.3.0",
		"sirv-cli": "0.4.5",
		"svelte-fullcalendar": "0.0.4"
	},
	"scripts": {
		"build": "rollup -c",
		"autobuild": "rollup -c -w",
		"dev": "run-p start:dev autobuild",
		"start": "sirv public --single",
		"start:dev": "sirv public --single --dev"
	}
}
```

But still there are warning like that at compile :
```
(!) Plugin svelte: FullCalendar has unused export property 'allDayHtml' (or 'schedulerLicenseKey', ..). If it is for external reference only, please consider using `export const 'allDayHtml'`
node_modules\svelte-fullcalendar\src\components\FullCalendar.svelte
129:     timeZoneParam = null,
130:     titleRangeSeparator = null,
```
Do we need to be worry about ?

**Thanks for the good work :D
Have a nice new year**

**Comment by YogliB** [comment]
Thanks for the feedback!
Can u please check if your problems were solved in the latest version (0.0.6)?

**Comment by tranKILL** [comment]
Seems to work fine on Windows 10 ;)
